helm_resource/helm-apply-helper.py

```diff
@@ -18,13 +18,17 @@
     flags.extend(['--set', '%s=%s' % (key, image)])
     continue
 
+  key0 = os.environ.get('TILT_IMAGE_KEY_REGISTRY_%s' % i, '')
   key1 = os.environ.get('TILT_IMAGE_KEY_REPO_%s' % i, '')
   key2 = os.environ.get('TILT_IMAGE_KEY_TAG_%s' % i, '')
-  parts = image.split(':')
-  repo = ':'.join(parts[:len(parts)-1])
-  tag = ':'.join(parts[len(parts)-1:])
-  flags.extend(['--set', '%s=%s' % (key1, repo),
-                '--set', '%s=%s' % (key2, tag)])
+  registry, image = image.split('/', 1)
```

Comment:
This won't work if there is no registry, which is a common setup for e.g. Docker Desktop or some minikube configurations:

```
   helloworld │ STEP 3/3 — Deploying
   helloworld │      Running cmd: python3 /Users/milas/dev/tilt-extensions/helm_resource/helm-apply-helper.py --create-namespace
   helloworld │      Traceback (most recent call last):
   helloworld │        File "/Users/milas/dev/tilt-extensions/helm_resource/helm-apply-helper.py", line 24, in <module>
   helloworld │          registry, image = image.split('/', 1)
   helloworld │      ValueError: not enough values to unpack (expected 2, got 1)
```

If you want to test locally w/ minikube:
```sh
minikube start --driver=docker --container-runtime=docker
./helm_resource/test/test.sh
```

Reply:
Yep, I figured this might be a problem for certain use-cases but hadn't run into it myself (as I usually use k3d). I'll look at how to handle this as well

Reply:
Should be all set now, I tested it on k3d and docker-desktop